CHANGES.md

```diff
@@ -1,18 +1,29 @@
 ringpop-go changes
 ==================
 
-v0.DEV (to be released)
------------------------
+v0.4.0
+------
 
-* Remove `File` and `Host`-based discover providers in favor of
-  `DiscoverProvider` interface #119
-* Ringpop will always assume the current host (`ringpop.node.identity`) is part
-  of the cluster. Previously, it was true for only `Host`-based discovery.
-* Add Go 1.6 testing on CI
+* Feature: Faulty nodes are now automatically reaped from nodes' membership
+  lists after 24 hours (by default). #123
+* New options for controlling suspect and reaping times. #123
+* Add new `Ready` and `Destroyed` events to ringpop. #125
+* Add additional logging to bring ringpop-go on par with ringpop-node log
+  messages. #116
+* Fix bug where Ringpop automatically added itself to the bootstrap hosts for
+  host-based bootstrapping, but not other bootstrapping methods. #120
+* Fix race condition where membership and hashring could be inconsistent with
+  each other. #112
+* Remove `File` and `Host` options from bootstrap options in favor of
+  `DiscoverProvider` interface. #120
+* Add Go 1.6 to testing on CI
```

Comment:
Looks like that in the end it's a matter of taste. As a consumer of a library, I prefer authors to be more verbose than scarce. Given that, the detail needs to be organized, otherwise it's useless too.

I`m not going to block this, since, like you showed, everything can be figured out quite easily. Now you know see my preferences. :-)

LGTM.